### PR TITLE
Jkantor/add file size

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Add file size to the  ESG file info response 

JIRA: [AU-381](https://openedx.atlassian.net/browse/AU-381)

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: openassessment/xblock/submission_mixin.py [hunk 1]
```diff
@@ -671,6 +672,7 @@ def get_files_info_from_user_state(self, username):
                             download_url=download_url,
                             description=description,
                             name=file_name,
+                            size=None,
```
Comment (mattcarter) [inline openassessment/xblock/submission_mixin.py:675]: Should we update the docstring to match?

File: openassessment/xblock/submission_mixin.py [hunk 2]
```diff
@@ -723,6 +725,7 @@ def get_all_upload_urls_for_user(self, username_or_email):
                         download_url=download_url,
                         description='',
                         name='',
+                        size=None,
```
Comment (mattcarter) [inline openassessment/xblock/submission_mixin.py:728]: Should we update the docstring to match?

File: openassessment/xblock/submission_mixin.py (hunk 1 above)
Comment (jansenk) [reply openassessment/xblock/submission_mixin.py:675]: Yeah, this is actually super outdated

**Review by nsprenkle — APPROVED** [review]
Vaguely worried by how many places this touches. Looks like you've caught all them but it would be worth some manual smoke testing if it hasn't been done already.

**Review by leangseu-edx — APPROVED** [review]
LGTM 👍

**Pull request merged**